src/domUpdates.js

```diff
@@ -24,15 +24,52 @@ const domUpdates = {
     </article>`);
   },
 
-  displayNavBar(user, navBar){
-    navBar.insertAdjacentHTML('afterbegin',
+  clearTrips(tripList) {
+    tripList.innerHTML = ''
+  },
+
+  displaySidebar(user, sidebar){
+    sidebar.insertAdjacentHTML('afterbegin',
     `<img src="./images/profpic.png" alt="user's profile picture">
     <h2>${user.name}</h2>
-    <h3>Welcome back, traveler!</h3>
-    <h3>Approved Trips: ${user.returnTripsByStatus('approved').length}
-      <br>Pending Trips: ${user.returnTripsByStatus('pending').length}
-      <br>In the past year you've spent $${user.getCostForYear()} on Travel</h3>
-    <button type="button" name="button">plan a trip!</button>`)
+    <div class="welcome-message">
+      <h3>Welcome back, traveler!</h3>
+      <h3>Approved Trips: ${user.returnTripsByStatus('approved').length}
+        <br>Pending Trips: ${user.returnTripsByStatus('pending').length}
+        <br>In the past year you've spent $${user.getCostForYear()} on Travel</h3>
+    </div>
+    <button class="plan-trip-button">plan a trip!</button>`);
+  },
+
+  displayNewTripForm(destinations) {
+    document.querySelector('.new-trip-form').insertAdjacentHTML('afterbegin',
+    `<select class="destination-list" name="">
+    </select>
+    <input type="date" class="trip-start" min="${new Date().toISOString().substring(0, 10)}">
+    <input type="date" class="trip-end" min="${new Date().toISOString().substring(0, 10)}">
+    <input type="number" class="num-travelers" value="1" min="1">
+    <h3></h3>
```

Comment:
Is this <h3> being used for styling? I could have missed where something is being inserted.

Reply:
h3 tag...

Reply:
Oh no this function needs to get deleted! I moved it to `index.html` and fixed the `<h3>` over there.